docs/dev/table/connect.md

```diff
@@ -164,21 +164,8 @@ CREATE TABLE MyUserTable (
   'connector.properties.zookeeper.connect' = 'localhost:2181',
   'connector.properties.bootstrap.servers' = 'localhost:9092',
 
-  -- specify the update-mode for streaming tables
-  'update-mode' = 'append',
-
   -- declare a format for this system
-  'format.type' = 'avro',
```

Comment:
Does Avro support derive schema from connector?

Reply:
No, we don't support yet. That's why I change the example to json which is simpler.